### ISSUE TITLE
zee channels on jio tv not working properly

### ISSUE THREAD

**Issue by shekhar771** [botallen/repository.botallen#306]
**Your Environment**
- Add-on name: Jiotv

- Kodi version: Kodi 19.3
- Operating system version: Windows 10, Android tv
- Using from country: India

the video feed for few channels are not working properly .the sound and video are not loading properly . not aa network issue.checked the official jiotv app it seems to work there.video is scrambling and not showing properly

**To Reproduce**
1. Go to jio tv
2. Click on zee tv or nat geo or multiple other channels 

**Screenshots**
![screenshot00001](https://user-images.githubusercontent.com/54745008/188315105-cdb609da-a4ed-41e8-a913-d489f2384879.png)
![screenshot00002](https://user-images.githubusercontent.com/54745008/188315107-2b5ed173-ad0c-47f3-aea5-9fffdf7fa288.png)
![screenshot00003](https://user-images.githubusercontent.com/54745008/188315111-1d1f66c4-a785-4f56-9d8f-e7f0b62d8b3e.png)

**Comment by shri1990** [comment]
Yes, most of the channels are not working and just plays the same content for 5-10seconds and gets stuck. the same channels see working in the mobile app.
@iMro0t, can you please check this?

**Comment by Rajeshsahoo12** [comment]
I have checked this on PC workaround. On kodi 19.4 application setup with latest botallen repo with jio tv add-on installed.

As many channels are not working. Specially related to ZEE. So, what I have read in past many issue opened.

Goto to JIO TV add-on settings as per below.

![image](https://user-images.githubusercontent.com/54216637/188507790-6c87bdd9-2ed2-41ef-9237-c82afd8deede.png)

Then goto MORE in left navigation panel.

![image](https://user-images.githubusercontent.com/54216637/188507866-7d7e8665-542e-45b6-b72e-bcb7fd5e2f83.png)

And disable EXTRA HD CHANNELS toogle to off.

Then just restart the kodi app & try running non working channels. Hope this helps.

Thanks


**Note :- Tried all the above steps on PC KODI 19.4 MATRIX ALONG WITH LATEST BOTALLEN REPO (2.0) & JIO TV ADD-ON (2.3.0)

![image](https://user-images.githubusercontent.com/54216637/188508124-a6e4af0e-2f8a-41ec-8a6d-74417a64480c.png)


![image](https://user-images.githubusercontent.com/54216637/188508149-7747e6c8-dc20-40e2-a13c-0309fc2d6c83.png)**
